### ISSUE TITLE
Make the currently Active Tab more distinguishable from others

### ISSUE THREAD

**Issue by LeonnardoVerol** [fabiospampinato/vscode-monokai-night#9]
Suggestion:
```
"tab.activeBackground": "#161616",
"tab.activeBorder": "#161616",
"tab.hoverBackground": "#161616",
```

Optional Tab Active Border:
```
"tab.activeBorder": "#3EA8E3",
```

**Comment by fabiospampinato** [comment]
I agree this is needed, perhaps we should make all `#1f1f1f` colors a bit brighter and then inactive tabs darker. Your suggestion would stop making the active tab blend with the current editor which is a nice effect.

**Comment by ImSeaWorld** [comment]
Hope this gets added soon, I tend to get lost in a sea of tabs.

**Comment by validatedev** [comment]
This is the theme and colors that I want, don't like blueish themes which are all dominating the VSCode theme extension market. But I don't like the color of the tab either. Please create a PR for that. Thank you.

**Comment by ImSeaWorld** [comment]
@fabiospampinato I've been using this for a few years now, is this going to be added or is a PR necessary?

**Comment by fabiospampinato** [comment]
Fixed in v1.6.2 🎉 I hope you'll like it.

Before:

<img width="1728" alt="Screen Shot 2023-01-31 at 21 54 38" src="https://user-images.githubusercontent.com/1812093/215893695-f8c88185-ce5e-4ad3-b7d4-660b1b060162.png">

After:

<img width="1728" alt="Screen Shot 2023-01-31 at 22 01 46" src="https://user-images.githubusercontent.com/1812093/215893708-50b1a285-07a4-46a5-82f8-9ecbefb58ad7.png">
